src/components/editor-page/editor-pane/hooks/yjs/use-awareness.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Awareness } from 'y-protocols/awareness'
+import { useEffect, useMemo } from 'react'
+import { addOnlineUser, removeOnlineUser } from '../../../../../redux/realtime/methods'
+import { ActiveIndicatorStatus } from '../../../../../redux/realtime/types'
+import { useApplicationState } from '../../../../../hooks/common/use-application-state'
+import type { Doc } from 'yjs'
+import { Logger } from '../../../../../utils/logger'
+
+const ownAwarenessClientId = -1
+
+interface UserAwarenessState {
+  user: {
+    name: string
+    color: string
+  }
+}
+
+const userColors = [
+  { color: '#30bced', light: '#30bced33' },
+  { color: '#6eeb83', light: '#6eeb8333' },
+  { color: '#ffbc42', light: '#ffbc4233' },
+  { color: '#ecd444', light: '#ecd44433' },
+  { color: '#ee6352', light: '#ee635233' },
+  { color: '#9ac2c9', light: '#9ac2c933' },
+  { color: '#8acb88', light: '#8acb8833' },
+  { color: '#1be7ff', light: '#1be7ff33' }
```

Comment:
Maybe it would be a good idea to generate the light color from the normal color automatically. At least in this map… We're probably want to make it possibly to define both independenantly so no changing in the usage of this map

Reply:
Not sure if this is necessary because the server should set the initial color of the client. This code here is only temporary. I added an todo.